borg/remote.py

```diff
@@ -191,6 +191,10 @@ def __exit__(self, exc_type, exc_val, exc_tb):
             self.rollback()
         self.close()
 
+    @property
+    def id_str(self):
+        return bin_to_hex(self.id)
```

Comment:
Hmmm, maybe just `__str__`? This ID is the only kind of string representation that would make sense for Manifest (and Repository)